ast.go

```diff
@@ -145,7 +145,12 @@ func (fa *ASTFunc) generateParameters() []ast.Expr {
 		if p.Type.IsSlice && !p.Type.IsReturnArgument {
 			hasDeclaration = true
 
-			fa.addGoToCSliceConversion(p.Name, p.Type)
+			
+			if _, ok := p.Heuristics["pass_c"]; !ok {
```

Comment:
I think I have not found one that was correct with a copy, but "clang_sortCodeCompletionResults" is a case where both version work because we do not need the original array. So it is not wrong, that is what I meant.